issues.md

```diff
@@ -2,13 +2,4 @@
 
 This document tracks issues with the conformance e2e suite and their resolutions.
 
-## e2e: Internet connection test should not rely on ICMP achievement
-
-* https://github.com/kubernetes/kubernetes/issues/51145  
-* https://github.com/kubernetes/kubernetes/pull/51287  
-* https://github.com/kubernetes/kubernetes/pull/53381  
-
-## ThirdParty resource e2e test should not be marked as conformance
-
-* https://github.com/kubernetes/kubernetes/issues/52822  
-* https://github.com/kubernetes/kubernetes/pull/52823  
+## At this time all known major issues have been fixed in the 1.8 e2e test suite.
```

Comment:
I'm glad that these are all fixed by I think it's worthwhile to keep a historical record, at least for a little while, to demonstrate that SIG-architecture has been working with program participants to deal with their issues.

Reply:
I'll update to denote [FiV-1.8].

Reply:
@dankohn In thinking about this it makes more sense to me to have a single tracking issue in this repo that other issues can reference.  Otherwise this file will require constant maintenance.

Reply:
e.g. - https://github.com/cncf/k8s-conformance/issues/27   

Let me know what you think and I'll fix up this PR to point to that tracking issue.

Reply:
+1